embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java

```diff
@@ -67,6 +67,11 @@
         @ConfigDefault("null")
         public Optional<String> getAccessKeyId();
 
+        @Config("http_proxy")
+        @ConfigDefault("null")
+        public Optional<HttpProxy> getHttpProxy();
+        public void setHttpProxy(Optional<HttpProxy> httpProxy);
```

Comment:
Okay. I will fix this by 3rd one.

As another idea, how about introducing new `EMBULK_HTTP_PROXY` env? If users can configure http proxy for Embulk as this en. The name is reasonable for us. Because we don't need to care of the compatibility issues so much.